PSOperationsTests/PSOperationsTests.swift

```diff
@@ -973,4 +1006,54 @@ class PSOperationsTests: XCTestCase {
         XCTAssertEqual(1, opQ.operationCount)
         XCTAssertTrue(op.waitCalled)
     }
+    
+    /*
+        In 9.1 (at least) we found that occasionaly OperationQueue would get stuck on an operation
+        The operation would be ready, not finished, not cancelled, and have no dependencies. The queue
+        would have no other operations, but the op still would not execute. We determined a few problems
+        that could cause this issue to occur. This test was used to invoke the problem repeatedly. While we've
+        seen the opCount surpass 100,000 easily we figured 25_000 operations executing one right after the other was
+        a sufficient test and is still probably beyond typical use cases. We wish it could be more concrete, but it is not.
+    */
+    func testOperationQueueNotGettingStuck() {
+        
+        var opCount = 0
+        var requiredToPassCount = 25_000
+        let q = OperationQueue()
+        
+        let exp = expectationWithDescription("requiredToPassCount")
+        
+        func go() {
+            
+            if opCount >= requiredToPassCount {
+                exp.fulfill()
+                return
+            }
+            
+            let blockOp = BlockOperation {
+                (finishBlock: Void -> Void) in
+                finishBlock()
+                go()
```

Comment:
Is there something inherent in the problem that makes a 1-by-1 approach better than pausing the queue, adding 25,000 items, and resuming the queue? If they both make the original problem occur, I'd say the 2nd approach may tighten the timing between op executions and let you see if you've really got it nailed. But if it's more prevalent with the 1-by-1 approach then absolutely don't change it.